packages/apps/storybook/.env

```diff
@@ -1,2 +1,2 @@
-# Registered client which have access to scopes "imodels:read", "imodels:modify", "projects:read", and "itwins:modify", redirectUrl should be http://localhost:6006/signin-oidc.html
+# Registered client which have access to scopes "imodels:read", "imodels:modify", "projects:read", "itwins:read" and "itwins:modify", redirectUrl should be http://localhost:6006/signin-oidc.html
```

Comment:
Unless you still call the project API somewhere, projects:read should not be required anymore.

Reply:
Done